mscore/downloadUtils.cpp

```diff
@@ -55,6 +56,15 @@ void DownloadUtils::download(bool showProgress)
       QObject::connect(&manager, SIGNAL(finished(QNetworkReply*)), this, SLOT(downloadFinished(QNetworkReply*)));
       QObject::connect(reply, SIGNAL(finished()), &loop, SLOT(quit()));
 
+      QTimer timer;
+      timer.setSingleShot(true);
+      timer.start(timeOutSecs);
+      timer.callOnTimeout([&reply]() {
```

Comment:
/home/travis/build/musescore/MuseScore/mscore/downloadUtils.cpp: In member function ‘void Ms::DownloadUtils::download(bool, int)’:
    /home/travis/build/musescore/MuseScore/mscore/downloadUtils.cpp:62:13: error: ‘class QTimer’ has no member named ‘callOnTimeout’
           timer.callOnTimeout([&reply]() {
                 ^

See https://doc.qt.io/qt-5/qtimer.html#callOnTimeout:
This function was introduced in Qt 5.12.

Potential fix:
This method is provided for convenience. It's equivalent to calling QObject::connect(timer, &QTimer::timeout, timer, slot, connectionType).

Reply:
Ah, I've missed that one with 5.9